src/modules/components/banners/notification/notification.docs.html

```diff
@@ -46,6 +51,10 @@ <h3 class="heading-small mb-m">Configuration</h3>
       <pdk-notification type="locked">
         The requested resource is not available. <a href="">Try again</a>
       </pdk-notification>
+
+      <pdk-notification type="confirmation" flow="true">
+        No more information required
```

Comment:
In the description I'd put something like:
This is an example of a notification with full flow... or something on those lines